### PR TITLE
Fix lab: dumber examples

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

File: panel/lab/components/fields/color/index.vue [hunk 1]
```diff
@@ -1,72 +1,148 @@
...
+			<k-lab-example label="Autofocus">
+				<k-color-field
+					:autofocus="true"
+					label="color"
+					:value="value"
+					@input="value = $event"
+				/>
+			</k-lab-example>
+
+			<k-lab-example label="Required">
```
Comment (bastianallgeier) [inline panel/lab/components/fields/color/index.vue:22]: There's an issue with the required state. I think I added some CSS at some point to highlight invalid inputs and the wrapper component unset it properly. Now it's no longer unset. 

Screenshot 2024-01-22 at 11 02 39

File: panel/lab/components/fields/object/index.vue [hunk 2]
```diff
@@ -3,24 +3,18 @@
...
 		<k-lab-example label="Default">
-			<k-object-field :fields="fields" label="Object" />
+			<k-object-field
+				:fields="[
+					{
+						label: 'Name',
+						name: 'name',
+						type: 'text'
+					}
+				]"
```
Comment (bastianallgeier) [inline panel/lab/components/fields/object/index.vue]: This was already wrong before. The fields need to be defined as an object.

File: panel/lab/components/fields/radio/index.vue [hunk 3]
```diff
@@ -1,3 +1,90 @@
 <template>
-	<k-lab-options-field-examples type="radio" />
+	<k-lab-form>
```
Comment (bastianallgeier) [inline panel/lab/components/fields/radio/index.vue:2]: Something is strange here and I don't know what it is, but the examples without columns don't have a proper width for each option.

File: panel/lab/components/inputs/timeoptions/index.vue [hunk 4]
```diff
@@ -1,7 +1,60 @@
...
+	data() {
+		return {
+			value: null
+		};
+	}
+};
+</script>
+
 <style>
 .k-lab-input-examples .k-timeoptions-input {
```
Comment (bastianallgeier) [inline panel/lab/components/inputs/timeoptions/index.vue]: Those styles are no longer applied

File: panel/lab/components/fields/radio/index.vue (hunk 3 above)
Comment (distantnative) [reply panel/lab/components/fields/radio/index.vue:2]: Is that different to the current live lab?

Comment (bastianallgeier) [reply panel/lab/components/fields/radio/index.vue:2]: You are right, it actually isn't

File: panel/lab/components/fields/color/index.vue (hunk 1 above)
Comment (bastianallgeier) [reply panel/lab/components/fields/color/index.vue:22]: I removed the outline. I think it's too global and not super helpful after all. When we look closer into validation in your PR, we might find better solutions to test this.

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**